src/docs/Components/Hero/index.mdx

```diff
@@ -47,7 +47,7 @@ This following case with a boxed content in the hero can become hard to understa
 
 </Highlight>
 
-<Preview path="box" nude/>
+<Preview path="boxed-content" nude />
```

Comment:
The box is under the text on the `medium` breakpoint

The cases are: 
- S & M ➡️ box under the text
- L ➡️ box under the text and on the right side of the text
- XL & XXL ➡️ box on the right side of the text

[Abstract collection](https://share.goabstract.com/293e5252-0102-4635-8188-2380ba6c7668?collectionLayerId=84c2c89e-3039-42e1-9e9c-b5a1818ae00d&mode=design)

Reply:
@nherwegh Normally this is already the case.
Can you confirm that this is good for you?

http://fix9456hero.demo.mozaic-design-system-dtrp.appspot.com/Components/Hero/#title-with-text-and-boxed-content

Reply:
I just corrected it. 
I'll let you check it and tell me if it's good.